src/components/StarCheckbox/index.tsx

```diff
@@ -21,14 +24,20 @@ const StarCheckbox: React.FunctionComponent<CheckboxProps> = ({
     return (
         <label className={wrapperClassnames}>
             <span className={styles.container}>
-                <input
-                    checked={checked}
-                    type="checkbox"
-                    onChange={(e: any) =>
-                        onChange ? onChange(e as CheckboxChangeEvent) : null
-                    }
-                    value={value}
-                />
+                <Tooltip
+                    title={checked ? "Remove highlight" : "Highlight"}
```

Comment:
would something like "outline agent" and "remove outline" be clearer? @blairlyons

Reply:
not sure, but I think highlight is ok. Since the selected agent also gets outlined (in different color) I don't think changing to outline will be much clearer?